.github/workflows/size-limit.yml

```diff
@@ -3,13 +3,14 @@ on:
   pull_request:
     branches:
       - main
+      - next
 jobs:
   size:
     runs-on: ubuntu-latest
     env:
       CI_JOB_NUMBER: 1
     steps:
       - uses: actions/checkout@v1
-      - uses: chanzuckerberg/size-limit-action@master
+      - uses: andresz1/size-limit-action@v1
```

Comment:
Trying out using the upstream action instead of our fork.

Seems our fork is doing stuff to prevent more than 1 size-limit comment per PR. Not sure the added complexity of a fork is worth worrying about doing that.